inlang/source-code/env-variables/src/api.ts

```diff
@@ -21,8 +21,10 @@ export const publicEnvVariablesSchema = z.object({
 		.url()
 		.regex(/^(?!.*\/$).+$/, "Must not end with a slash")
 		.describe("The base url of the server e.g. https://inlang.com"),
+	PUBLIC_ALLOWED_AUTH_URLS: z
+		.string()
+		.describe("List of allowed base urls eg https://inlang.com,https://manage.inlang.com"),
```

Comment:
@janfjohannes do we also have the domains we need locally (localhost:3000 and localhost:4004) or do I need to add them again in Doppler?

Reply:
no need for anything manual locally it should all just work